src/pages/organize/[orgId]/people/lists/[viewId]/shared.tsx

```diff
@@ -42,9 +45,16 @@ async function getAccessLevel(
     return 'configure';
   }
 
-  const accessList = await apiClient.get<ZetkinObjectAccess[]>(
-    `/api/orgs/${orgId}/people/views/${viewId}/access`
-  );
+  let accessList: ZetkinObjectAccess[] = [];
+  try {
+    accessList = await apiClient.get<ZetkinObjectAccess[]>(
+      `/api/orgs/${orgId}/people/views/${viewId}/access`
+    );
+  } catch (e) {
+    if ((e as Error).message == 'Error during request: 403') {
+      return null;
+    }
```

Comment:
This try-catch is great! But I think that the `if` is not neccessary, because whatever error appears we want this function to return `null`